lib/foreman/renderer.rb

```diff
@@ -236,21 +240,37 @@ def dns_lookup(name_or_ip)
       raise e
     end
 
+    def load_hosts(search: '', includes: nil)
```

Comment:
I believe we should not add this here. Does it make sense to have this method e.g. in a provisioning template? This method will be available in the host object as well. `Host.first.load_hosts` does not make a lot of sense. Luckily #5683 will provide an easy way to make this possible.

Reply:
I agree, once the refactoring is done, I'd create a new scope for reporting templates (or other ways around, based on what gets in first)